tests/bluetooth/init/prj_ctlr_4_0_dbg.conf

```diff
@@ -25,6 +25,7 @@ CONFIG_BT_CTLR_CONN_RSSI=n
 CONFIG_BT_CTLR_ADV_INDICATION=n
 CONFIG_BT_CTLR_SCAN_REQ_NOTIFY=n
 CONFIG_BT_CTLR_SCAN_REQ_RSSI=n
+CONFIG_BT_CTLR_OPTIMIZE_FOR_APP_DEFAULT=y
```

Comment:
It's unclear why these specific 3 .conf files have these specific values set. 

Please expand the commit message to explain.

Reply:
These are added as coverage in CI for newly added KConfigs.... New  Kconfig are generally covered either in samples.yaml or in tests/bluetooth/init  builds.

Reply:
Okay, thanks, but please put that in the commit message :)

Reply:
added as separate commit.